Gordon360/Models/CCT/FINALIZATION_MARK_AS_CURRENTLY_COMPLETEDResult.cs

```diff
@@ -1,21 +0,0 @@
-﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
-using System;
-using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations.Schema;
-
-namespace Gordon360.Models.CCT
-{
-    public partial class FINALIZATION_MARK_AS_CURRENTLY_COMPLETEDResult
-    {
-        public string UserID { get; set; }
-        public string Period { get; set; }
-        public bool FinalizationCompleted { get; set; }
-        public string RootQuery { get; set; }
-        public string BypassApprover { get; set; }
-        public DateTime? DateFinalized { get; set; }
-        public string IgnoreHoldsApprover { get; set; }
-        public DateTime DateInserted { get; set; }
-        public DateTime DateUpdated { get; set; }
-        public string HeadcountsAsOfDateFinalized { get; set; }
-    }
-}
```

Comment:
@EjPlatzer is this one of the files I need? I've been reverting it constantly manually.

Reply:
Yes, this file is needed. If you look at the diff for this commit, you'll notice that it wasn't actually, deleted, it was renamed (because of the schema separation), and also all of the fields were removed. I'm not sure why, when you run EF Core refresh, it doesn't map the fields.

Reply:
It's actually because I manually reverted the delete then moved it into the new folder. [b5e1191](https://github.com/gordon-cs/gordon-360-api/pull/733/commits/b5e11913a504f3bb4fb03826b81cb8dfc82b3d9e)

Reply:
Yes, I see that you reverted it now. I was responding to this comment on the commit where it was first changed (before you changed it back). Seems like it's fixed now.

I'm not sure I understand what you're confused about re: files to include for EF Core updates - you should not commit changes to models if you didn't change the database those models are derived from (except for cases where you're changing some EF Core setting that affects them).